examples/NWB-Allen-Institute-Example.ipynb

```diff
@@ -0,0 +1,1464 @@
+{
```

Comment:
I would not push NB in git.
I propose instead when possible to use script + sphinx-gallery that generate NB

Reply:
I agree, NB can become quite big if they include images and are updated each time they are executed.

Reply:
removed